RetrospectiveExtension.Frontend/components/__tests__/feedbackItem.test.tsx

```diff
@@ -0,0 +1,261 @@
+  columnProperties:
+  {
+    id: testColumnUuidOne,
+    title: 'Test Feedback Column One',
+    iconClass: 'far fa-smile',
+    accentColor: '#008000',
+  },
+  columnItems:
+    [
+      {
+        feedbackItem: testFeedbackItem,
+        actionItems: []
+      },
+    ]
+};
+testColumnsObj[testColumnUuidTwo] = {
+  columnProperties:
+  {
+    id: TooltipOverflowMode,
+    title: testColumnTwoTitle,
+    iconClass: 'far fa-smile',
+    accentColor: '#008100',
+  },
+  columnItems: []
+};
+const testColumns = mocked(testColumnsObj);
+
+const testColumnProps = mocked({
+  columns: testColumns,
+  columnIds: testColumnIds,
+  columnName: testColumns[testColumnUuidOne].columnProperties.title,
+  columnId: testColumnUuidOne,
+  accentColor: testColumns[testColumnUuidOne].columnProperties.accentColor,
+  iconClass: testColumns[testColumnUuidOne].columnProperties.iconClass,
+  workflowPhase: WorkflowPhase.Act,
+  isDataLoaded: false,
+  columnItems: testColumns[testColumnUuidOne].columnItems,
+  team: {
+    id: uuid(),
+    identity:{
+      customDisplayName:'Test Web API Identity Custom Display Name',
+      descriptor:{
+        identifier:'Test Identifier',
+        identityType:'Test Identity Type'
+      },
+      id: uuid(),
+      isActive: true,
+      isContainer: false,
+      masterId: uuid(),
+      memberIds:[],
+      memberOf:[],
+      members:[],
+      metaTypeId:5,
+      properties:[],
+      providerDisplayName:'Test Web API Identity Provider Display Name',
+      resourceVersion:10,
+      socialDescriptor:'Test Social Descriptor',
+      subjectDescriptor:'Test Subject Descriptor',
+      uniqueUserId:500,
+    },
+    name: 'Test Web API Team Name',
+    description: 'Test Web API Team Description',
+    identityUrl: '',
+    projectId: uuid(),
+    projectName: 'Test Azure DevOps Retrospectives Extension',
+    url: ''
+  },
+  boardId: testBoardId,
+  boardTitle: 'Test Feedback Board',
+  defaultActionItemIteration: testTeamId,
+  defaultActionItemAreaPath: testTeamId,
+  nonHiddenWorkItemTypes: [testWorkItemType],
+  allWorkItemTypes: [testWorkItemType],
+  isBoardAnonymous: false,
+  shouldFocusOnCreateFeedback: false,
+  hideFeedbackItems: false,
+  onVoteCasted: jest.fn(() => { }),
+  addFeedbackItems: jest.fn((
+    columnId, columnItems, shouldBroadcast, newlyCreated, showAddedAnimation, shouldHaveFocus, hideFeedbackItems) => {
+
+  }),
+  removeFeedbackItemFromColumn: jest.fn((
+    columnIdToDeleteFrom, feedbackItemIdToDelete, shouldSetFocusOnFirstAvailableItem) => { }),
+  refreshFeedbackItems: jest.fn((feedbackItems, shouldBroadcast) => { }),
+});
+
+describe('Feedback Item', () => {
+  test('Render a Feedback Item with no child Feedback Items.', () => {
+    const testProps = FeedbackColumn.createFeedbackItemProps(
+      testColumnProps, testColumnItem, true);
+
+    const wrapper = shallow(<FeedbackItem {...testProps} />);
+    const component = wrapper.children().dive();
```

Comment:
so after I merged in the application insights changes from dev, all of my expects started failing. adding this .dive() call resolved the issue - the `withAITracking` wrapper added to the components makes enzyme think the wrapper is the top level component, so the shallow render believes the first level of children is the `FeedbackItem`.